`NullPointerException` Attempt to invoke virtual method `void com.machine.thee.presentation.SecondaryDisplay.dismiss()` on a null object reference

When calling `terminate` method before using any other methods, the `display` is null.


```
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
    at com.getcapacitor.Bridge.lambda$callPluginMethod$0(Bridge.java:830)
    at com.getcapacitor.Bridge.$r8$lambda$ehFTi5f4HhVNFKTbCKAYDkpQYRA(Unknown Source:0)
    at com.getcapacitor.Bridge$$ExternalSyntheticLambda3.run(Unknown Source:8)
    at android.os.Handler.handleCallback(Handler.java:942)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:201)
    at android.os.Looper.loop(Looper.java:288)
    at android.os.HandlerThread.run(HandlerThread.java:67)
Caused by: java.lang.reflect.InvocationTargetException
    at java.lang.reflect.Method.invoke(Native Method)
    at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:138)
    at com.getcapacitor.Bridge.lambda$callPluginMethod$0(Bridge.java:821)
    at com.getcapacitor.Bridge.$r8$lambda$ehFTi5f4HhVNFKTbCKAYDkpQYRA(Unknown Source:0) 
    at com.getcapacitor.Bridge$$ExternalSyntheticLambda3.run(Unknown Source:8) 
    at android.os.Handler.handleCallback(Handler.java:942) 
    at android.os.Handler.dispatchMessage(Handler.java:99) 
    at android.os.Looper.loopOnce(Looper.java:201) 
    at android.os.Looper.loop(Looper.java:288) 
    at android.os.HandlerThread.run(HandlerThread.java:67) 
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.machine.thee.presentation.SecondaryDisplay.dismiss()' on a null object reference
    at com.machine.thee.presentation.CapacitorPresentationPlugin.terminate(CapacitorPresentationPlugin.java:67)
    at java.lang.reflect.Method.invoke(Native Method) 
    at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:138) 
    at com.getcapacitor.Bridge.lambda$callPluginMethod$0(Bridge.java:821) 
    at com.getcapacitor.Bridge.$r8$lambda$ehFTi5f4HhVNFKTbCKAYDkpQYRA(Unknown Source:0) 
    at com.getcapacitor.Bridge$$ExternalSyntheticLambda3.run(Unknown Source:8) 
    at android.os.Handler.handleCallback(Handler.java:942) 
    at android.os.Handler.dispatchMessage(Handler.java:99) 
    at android.os.Looper.loopOnce(Looper.java:201) 
    at android.os.Looper.loop(Looper.java:288) 
    at android.os.HandlerThread.run(HandlerThread.java:67) 
```

--- a/android/src/main/java/com/machine/thee/presentation/CapacitorPresentationPlugin.java
+++ b/android/src/main/java/com/machine/thee/presentation/CapacitorPresentationPlugin.java
@@ -64,6 +64,10 @@ public class CapacitorPresentationPlugin extends Plugin {
 
   @PluginMethod
   public void terminate() {
+    if(display == null) {
+      return;
+    }
+
     display.dismiss();
   }
 

